frontend/src/index.tsx

```diff
@@ -13,6 +13,8 @@ import { AppRoute } from '~/libs/enums/enums.js';
 import { store } from '~/libs/packages/store/store.js';
 import { Auth } from '~/pages/auth/auth.js';
 
+import { ProtectedRoute } from './libs/components/protected-route/protected-route.js';
```

Comment:
```suggestion
import { ProtectedRoute } from './libs/components/components.js';
```

Reply:
Last time i have imported  Protected component to /libs/components/components.js'; and took it from there. And i got it as bug. Which way is true?

Reply:
It was in previous branch. I din't know that forbiden create new ones. So i created new